.travis.yml

```diff
@@ -36,22 +61,7 @@ before_install:
   - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - yarn install --no-lockfile
```

Comment:
I removed this because `ember try:one` (when `useYarn: true` is specified in config) already does `yarn install --no-lockfile --ignore-engines` when it is installing the scenario.

Reply:
if it is runnin `yarn install --no-lockfile` and `node_modules` already exists, are you sure it updates all deps to the latest possible versions?

Reply:
No, I'm unsure.  I'll add an explicit job to the first stage that is explicity testing transitive deps with the default set ("current" ember-source).

Reply:
OK, updated to add a single `--no-lockfile` job to the first stage. This way we don't have to ignore the lockfile for all jobs, and hopefully with this setup we can easily identify failures due to transient deps vs failures due to _actual_ repo changes.